cmd/edgectl/aes_install.go

```diff
@@ -987,24 +1009,35 @@ Removing the CRDs will cause your existing Ambassador Mappings and other resourc
 
 $ kubectl delete crd -l product=aes
 
-Aborting the installer to avoid corrupting an existing (but undetected) installation.
+The installer will now quit to avoid corrupting an existing (but undetected) installation.
 `
 
-const seeDocs = "See https://www.getambassador.io/docs/latest/tutorials/getting-started/"
+const seeDocs    = "See https://www.getambassador.io/docs/latest/tutorials/getting-started/"
+const seeDocsURL = "https://www.getambassador.io/docs/latest/tutorials/getting-started/"
+
+const phoneHomeDisabled  = "[aesInstall] Info: phone-home is disabled by environment variable"
+const installAndTraceIDs = "[aesInstall] Info: install_id = %s; trace_id = %s"
```

Comment:
Why the `[aesInstall]` prefix here? We don't use this anywhere else.

Reply:
it was meant to be a parallel construct to the installer.SetMetadatum and Report,  which say:
"[Metrics]..."
but I can remove it of course.

Reply:
The `[Metrics]` prefix et al are not seen by the user (except in verbose mode, which is really just "log to the screen" mode -- it could use a lot of refinement).

Reply:
Removed in this PR.